hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -577,6 +592,11 @@ private KeyInfo getProtobuf(boolean ignorePipeline, String fullKeyName,
         .setObjectID(objectID)
         .setUpdateID(updateID)
         .setParentID(parentObjectID);
+
+    FileChecksumProto fileChecksumProto = OMPBHelper.convert(fileChecksum);
```

Comment:
Could check fileChecksum null before call the convert.

Reply:
nullity is checked inside OMPBHelper.convert()